### ISSUE TITLE
Feature Request: Optohybrid Firmware Release Register

### ISSUE THREAD

**Issue by bdorney** [cms-gem-daq-project/OptoHybridv3#3]
Please change OHv3 firmware release info registers to match `CTP7` registers.  e.g. for the CTP7 we can see easily the release `vX.Y.Z` where `X` is the Major, `Y` is the Minor, and `Z` is the build, and the date in four dedicated registers:

```
eagle60 > kw RELEASE
0x6640000c None         GEM_AMC.GEM_SYSTEM.RELEASE
0x6640000c r    GEM_AMC.GEM_SYSTEM.RELEASE.MAJOR                        0x00000003
0x6640000c r    GEM_AMC.GEM_SYSTEM.RELEASE.MINOR                        0x00000002
0x6640000c r    GEM_AMC.GEM_SYSTEM.RELEASE.BUILD                        0x00000007
0x66400010 r    GEM_AMC.GEM_SYSTEM.RELEASE.DATE                         0x20171211
```

However the OH FW gives just the date as three separate registers:

```
0x65000004 None         GEM_AMC.OH.OH0.FPGA.CONTROL.RELEASE
0x65000008 r    GEM_AMC.OH.OH0.FPGA.CONTROL.RELEASE.YEAR                0x00002018
0x65000008 r    GEM_AMC.OH.OH0.FPGA.CONTROL.RELEASE.MONTH               0x00000001
0x65000008 r    GEM_AMC.OH.OH0.FPGA.CONTROL.RELEASE.DAY                 0x00000016
```

Definitely prefer the approach used by the CTP7.  Could we change the OH to have this layout?

**Comment by andrewpeck** [comment]
Actually I changed this already the other day 👍

It will be in the next version..

**Comment by andrewpeck** [comment]
Fixed in 2f30196fca0096dab45a9df18a26f22c2fd34c3a
